src/controllers/verifyCommand.ts

```diff
@@ -13,6 +13,8 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
+  const verificationString =
```

Comment:
let me know if there is any constant file to move the `verification string`.

Reply:
We can use the `src/constants/responses` 👍🏻

Reply:
resolved